### PR TITLE
added descriptions to AudioEffect, AudioEffectAmplify, and AudioEffec…

### PR DESCRIPTION
**Pull request by aarroz**
…tChorus.
I wrote in descriptions for three audio classes.

### CODE REVIEW

File: doc/base/classes.xml [hunk 1]
```diff
@@ -6083,14 +6085,17 @@
...
 	</brief_description>
 	<description>
+		Amplifies the volume of an audio source. Increase gain of the audio being routed through the bus.
 	</description>
 	<methods>
 		<method name="get_volume_db" qualifiers="const">
 			<return type="float">
 			</return>
 			<description>
+				Get's the set maximum volume.
```
Comment (Noshyaar) [inline doc/base/classes.xml]: Nitpicking :P

**Comment by aarroz** [comment]
Dang it. Fixed.

File: doc/base/classes.xml [hunk 2]
```diff
@@ -6083,14 +6085,17 @@
...
 	</brief_description>
 	<description>
+		Amplifies the volume of an audio source. Increase gain of the audio being routed through the bus.
 	</description>
 	<methods>
 		<method name="get_volume_db" qualifiers="const">
 			<return type="float">
 			</return>
 			<description>
+				Returns the set maximum volume.
```
Comment (Maxeonyx) [inline doc/base/classes.xml]: I think for this sprint we are not changing setters and getters because they are going to be removed

Comment (aarroz) [reply doc/base/classes.xml]: Should I just remove the descriptions for those methods then? I already wrote them before I was aware of that.

**Comment by Noshyaar** [comment]
Changes look okay for merging. Please squash commits together.

**Pull request merged**